### PR TITLE
Dereference environment alias in collector

### PR DESCRIPTION
**Pull request by atward**
Fixes c6c8f9c8093d074c2a7aa4b3cae60b7d43715777

everything was previously using environment_as_dict.

### CODE REVIEW

File: bespin/collector.py [hunk 1]
```diff
@@ -159,6 +159,7 @@ def convert_stack(path, val):
             env = configuration[["environments", environment]]
             if isinstance(env, six.string_types):
                 environment_as_dict = configuration[["environments", env]].as_dict()
+                env = configuration["environments", env]
```
Comment (delfick) [inline bespin/collector.py]: ah yeah, makes sense.

Though you need to pass a dictionary into the `__getitem__` syntax here.

    env = configuration[["environmets", env]]

Comment (atward) [reply bespin/collector.py]: Originally thought that, but it also supports tuple.
https://github.com/delfick/bespin/pull/69/files/455a53cf201870d72f42dc1a588942c4ccc4935d#diff-721a175167557506a913d8891aa8c9d8R169

Comment (delfick) [reply bespin/collector.py]: yeah, either way you can't pass multiple things into it

Comment (delfick) [reply bespin/collector.py]: oh, I see, python changes that into a tuple for you.... I'd rather be explicit to be honest

Comment (atward) [reply bespin/collector.py]: https://github.com/delfick/option_merge/blob/master/option_merge/merge.py#L199
https://github.com/delfick/option_merge/blob/master/option_merge/merge.py#L410

**Comment by atward** [comment]
I take it the pushed changes address your concerns.

**Comment by delfick** [comment]
yeap

**Pull request merged**